applications/ssotap/values-idfdev.yaml

```diff
@@ -1,4 +1,21 @@
 cadc-tap:
+  config:
+    uwsDatabaseUrl: "postgresql://localhost:5432/ssotap"
+    uwsDatabaseUsername: "ssotap"
+    pg:
+      image:
+        repository: "ghcr.io/lsst-sqre/tap-postgres-service"
+        pullPolicy: "Always"
+        tag: "1.16.0"
+
+  uws:
+    enabled: false
+
```

Comment:
Similarly here, we can assume that if CloudSQL is enabled, UWS is always disabled.

Reply:
Yep, removed this extra param. Should make the values more concise